vignettes/BiostringsQuickOverview.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Biostrings Quick Overview"
+author:
+- |
+  Hervé Pagès\
+  Fred Hutchinson Cancer Research Center\
+  Seattle, WA
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output:
+  html_document:
+  toc: yes
+  df_print: paged
+  number_sections: yes
+---
+
+Most but not all functions defined in the [Biostrings]{.sans-serif}
+package are summarized here.
+
+::: center
+::: {#table:Low_level_manipulation}
+  **Function**                       **Description**
+  ---------------------------------- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
This table as well as the miscellaneous don't follow the same format as the other tables. One of the reasons why we're converting to .Rmd is to make documentation easier to maintain by making it more readable. I would suggest changing the header this way:

```
Function | Description
---|---
```

For the rest of the table (lines 23-34), I would place a `|` between the left and right sides. For example,
```
  `length` | Return the number of sequences in an object.
```